{
    "issue": {
        "title": "[C++][CI] AppVeyor failures on s3fs-test",
        "body": "***Note**: This issue was originally created as [ARROW-16722](https://issues.apache.org/jira/browse/ARROW-16722). Please see the [migration documentation](https://gist.github.com/toddfarmer/12aa88361532d21902818a6044fda4c3) for further details.*\n\n### Original Issue Description:\nThose are probably due to a new Minio version:\r\nhttps://ci.appveyor.com/project/ApacheSoftwareFoundation/arrow/builds/43735193/job/0f1ocw3wqr6sb0vx\r\n\r\n```Java\n\r\n[ RUN      ] TestS3FSGeneric.CreateDir\r\nC:/projects/arrow/cpp/src/arrow/filesystem/test_util.cc(238): error: Failed\r\nExpected 'fs->CreateDir(\"AB/def/EF/GH\", true )' to fail with IOError, but got OK\r\n[       OK ] TestS3FSGeneric.CreateDir (2208 ms)\r\n```\r\n\r\n```Java\n\r\n[ RUN      ] TestS3FS.DeleteDirContentsAsync\r\nC:/projects/arrow/cpp/src/arrow/filesystem/s3fs_test.cc(889): error: Failed\r\nExpected '_fut.status()' to fail with IOError, but got OK\r\n[  FAILED  ] TestS3FS.DeleteDirContentsAsync (837 ms)\r\n```\r\n\r\n(note also the flurry of errors when trying to delete the Minio temp directory at test shutdown...)",
        "created_at": "2022-06-02T08:36:10.000Z",
        "updated_at": "2022-06-02T23:11:24.000Z",
        "labels": [
            "Migrated from Jira",
            "Component: C++",
            "Component: Continuous Integration",
            "Type: bug"
        ],
        "closed": true,
        "closed_at": "2022-06-02T23:11:24.000Z"
    },
    "comments": [
        {
            "created_at": "2022-06-02T08:41:21.065Z",
            "body": "***Note**: [Comment](https://issues.apache.org/jira/browse/ARROW-16722?focusedCommentId=17545347) by Antoine Pitrou (apitrou):*\nThere are similar errors on MinGW:\r\nhttps://github.com/apache/arrow/runs/6704811930?check_suite_focus=true\r\n"
        },
        {
            "created_at": "2022-06-02T08:45:05.838Z",
            "body": "***Note**: [Comment](https://issues.apache.org/jira/browse/ARROW-16722?focusedCommentId=17545350) by Ra\u00fal Cumplido (raulcd):*\nI think that these 4 failures for the latest nightly tests builds are related to this issue:\r\n- test-debian-11-python-3\n  \u00a0 <https://github.com/ursacomputing/crossbow/runs/6701775809>\n- test-fedora-35-python-3\n  \u00a0 <https://github.com/ursacomputing/crossbow/runs/6701845453>\n- test-ubuntu-20.04-python-3\n  \u00a0 <https://github.com/ursacomputing/crossbow/runs/6702169395>\n- test-ubuntu-22.04-cpp\n  \u00a0 <https://github.com/ursacomputing/crossbow/runs/6701496177?check_suite_focus=true>"
        },
        {
            "created_at": "2022-06-02T08:45:26.490Z",
            "body": "***Note**: [Comment](https://issues.apache.org/jira/browse/ARROW-16722?focusedCommentId=17545351) by Antoine Pitrou (apitrou):*\nGiven that we regularly get errors on random Minio upgrades, perhaps we should install a fixed Minio version for our tests, such as \"minio.RELEASE.2022-05-19T18-20-59Z\".\r\n\r\nSee <https://dl.min.io/server/minio/release/linux-amd64/archive/> , <https://dl.min.io/server/minio/release/windows-amd64/archive/> , etc.\r\n\r\ncc `[~raulcd]` \u00a0 `[~kou]` \u00a0 `[~assignUser]` \u00a0"
        },
        {
            "created_at": "2022-06-02T13:40:49.067Z",
            "body": "***Note**: [Comment](https://issues.apache.org/jira/browse/ARROW-16722?focusedCommentId=17545474) by Antoine Pitrou (apitrou):*\n`[~raulcd]` Would you like to take this up? Ideally we should make CI green again quickly :-)"
        },
        {
            "created_at": "2022-06-02T23:11:24.196Z",
            "body": "***Note**: [Comment](https://issues.apache.org/jira/browse/ARROW-16722?focusedCommentId=17545682) by Kouhei Sutou (kou):*\nIssue resolved by pull request 13299\n<https://github.com/apache/arrow/pull/13299>"
        }
    ]
}